### PR TITLE
Add new function add_ip()

### PR DESCRIPTION
**Pull request by roniezr**
Add new function add_ip() + two tests

### CODE REVIEW

File: rrmngmnt/network.py [hunk 1]
```diff
@@ -646,14 +646,32 @@ def if_down(self, nic, tcp_timeout=20, io_timeout=20):
...
+        Args:
+            nic (str): Interface name
+            ip (str): IP address to add
+            mask (str): IP netmask
+
+        Returns:
+            bool: True if add IP was success, False otherwise
+        """
+        cmd = "ip address add {ip}/{mask} dev {nic}".format(
+            ip=ip, mask=mask, nic=nic
```
Comment (myakove) [inline rrmngmnt/network.py]: one line

File: rrmngmnt/network.py [hunk 2]
```diff
@@ -646,14 +646,32 @@ def if_down(self, nic, tcp_timeout=20, io_timeout=20):
...
+        )
+        rc, _, _ = self.host.run_command(command=shlex.split(cmd))
```
Comment (myakove) [inline rrmngmnt/network.py]: return not bool(self.host.run_command(command=shlex.split(cmd))[0])

File: rrmngmnt/network.py (hunk 1 above)
Comment (roniezr) [reply rrmngmnt/network.py]: With one line I got pep8 error as followed:
rrmngmnt/network.py:669:80: E501 line too long (86 > 79 characters)

**Comment by lukas-bednar** [comment]
Thank you!

**Pull request merged**